avaframe/com1DFA/DFAfunctionsCython.pyx

```diff
@@ -301,7 +302,11 @@ def computeForceC(cfg, particles, fields, dem, dT, int frictType):
       if uMag<velMagMin:
         uMag = velMagMin
       forceBotTang = - areaPart * tau
-      forceFrict[j] = forceFrict[j] - forceBotTang/uMag
+      if explicitFriction>0:
```

Comment:
sugggest: if explicitFriction == 1 to be consitent with the comments in the other docstrings